mlflow/server/js/src/experiment-tracking/components/ExperimentView.js

```diff
@@ -89,6 +92,8 @@ export class ExperimentView extends Component {
       showFilters: false,
       showOnboardingHelper: onboardingInformationStore.getItem('showTrackingHelper') === null,
       searchInput: props.searchInput,
+      orderByKey: props.orderByKey,
+      orderByAsc: props.orderByAsc,
```

Comment:
Is there a reason we need to fork the state for `orderByKey` and `orderByAsc` here? can the ExperimentPage(the parent component) be the owner for this property? What do you think?

Reply:
Indeed there was no good reason. Removed it now!